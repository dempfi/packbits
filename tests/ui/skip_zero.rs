use packbits as _;

#[packbits::pack(bytes = 1)]
struct Bad {
    #[skip(0)]
    a: u8,
}

fn main() {}
