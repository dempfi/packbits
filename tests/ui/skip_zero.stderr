error: `skip` must be > 0
 --> tests/ui/skip_zero.rs:5:3
  |
5 |   #[skip(0)]
  |   ^
